source/Ship.h

```diff
@@ -460,6 +462,8 @@ class Ship : public Body, public std::enable_shared_from_this<Ship> {
 	bool isSpecial = false;
 	bool isYours = false;
 	bool isParked = false;
+	// Whether this ship is carried at the moment.
+	bool isCarried = false;
```

Comment:
Is it needed to have this variable? I would expect that there is already other code in the game that checks if a ship is carried and such code was then able to determine if a ships was carried without the need for this variable.